FoodTracker/Controllers/MealTableViewController.swift

```diff
@@ -0,0 +1,146 @@
+//
+//  MealTableViewController.swift
+//  FoodTracker
+//
+//  Created by BrainX IOS 4 on 2020-12-14.
+//
+import os.log
+import UIKit
+
+class MealTableViewController: UIViewController {
+    
+    //MARK: Outlets
+    @IBOutlet var tableView: UITableView!
+    
+    //MARK: Properties
+    var meals = [Meal]()
+    
+    //MARK: ViewController Lifecycle Methods
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        tableView.delegate = self
+        navigationItem.leftBarButtonItem = editButtonItem
+        loadData()
+    }
+
+    // MARK: Navigation Methods
+    override func prepare(for segue: UIStoryboardSegue, sender: Any?) {
+        super.prepare(for: segue, sender: sender)
+        
+        switch(segue.identifier) {
+        case "AddItem":
+            os_log("Adding a new meal.", log: OSLog.default, type: .debug)
+        case "showDetail":
+            guard let mealDetailViewController = segue.destination as? MealViewController else {
+                fatalError("Unexpected Controller: \(segue.destination)")
+            }
+            guard let mealTableViewCell = sender as? MealTableViewCell else {
+                fatalError("Unexpected Cell: \(sender ?? "")")
+            }
+            guard let indexPath = tableView.indexPath(for: mealTableViewCell) else {
+                fatalError("Selected cell is not in table.")
+            }
+            let selectedMeal = meals[indexPath.row]
+            mealDetailViewController.meal = selectedMeal
+        default:
+            fatalError("Unexpected Segue Identifier; \(segue.identifier ?? "")")
+        }
+    }
+    
+    //MARK: Actions
+    @IBAction
+    func unwindToMealList(sender: UIStoryboardSegue) {
+        if let sourceViewController = sender.source as? MealViewController, let meal = sourceViewController.meal {
+            if let selectedIndexPath = tableView.indexPathForSelectedRow {
+                meals[selectedIndexPath.row] = meal
+                tableView.reloadRows(at: [selectedIndexPath], with: .none)
+            } else {
+                let NewIndexPath = IndexPath(row: meals.count, section: 0)
```

Comment:
The convention for properties and other variables names is lowerCamel case while for some type (Class, Struct, TypeAlias etc) it should be UpperCamel case.